templates/default/sql/dml/undo/1.3.0__undo.sql.erb

```diff
@@ -4,3 +4,9 @@ DELETE FROM `hopsworks`.`variables` WHERE `id`='hdfs_log_storage_policy';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='service_discovery_domain';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='hopsworks_enterprise';
 DELETE FROM `hopsworks`.`variables` WHERE `id`='kubernetes_installed';
+DELETE FROM `hopsworks`.`variables` WHERE `id`='yarn_runtime';
+REPLACE INTO variables VALUES ("anaconda_dir", "<%= node['conda']['base_dir'] %>");
+DELETE FROM `hopsworks`.`variables` WHERE `id`='docker_mounts';
+REPLACE INTO variables VALUES ("anaconda_dir", "<%= node['conda']['base_dir'] %>");
```

Comment:
double replace for `anacond_dir`, also please be consistent and use `hopsworks`.`variables`.

Reply:
Fixing it. 
It is also broken by the new collumn that have been added to the variables table.